specs/stork-scheduler.yaml

```diff
@@ -101,11 +101,9 @@ spec:
         - /usr/local/bin/kube-scheduler
         - --address=0.0.0.0
         - --leader-elect=true
-        - --scheduler-name=stork
-        - --policy-configmap=stork-config
-        - --policy-configmap-namespace=kube-system
+        - --config=/etc/kubernetes/stork-config.yaml
         - --lock-object-name=stork-scheduler
```

Comment:
Guess, we don't need `--leader-elect=true` and `--lock-object-name=stork-scheduler` anymore, since these flags are set in configuration